src/tasks/platforms/common-platform-tasks.ts

```diff
@@ -72,4 +75,19 @@ export namespace CommonPlatformTasks {
     })
   }
 
+  export function oAuthProvidersExists(flags: any): Listr.ListrTask {
+    let kube = new KubeHelper(flags)
+    return {
+      title: 'Verify the existence of identity providers.',
+      enabled: () => isOpenshiftPlatformFamily(flags.platform) && flags['os-oauth'],
```

Comment:
I think checking `flags['os-oauth']` is not sufficient. 
Use can set this flag in CR also.  We need something similar to
https://github.com/che-incubator/chectl/blob/master/src/commands/server/start.ts#L219